install.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+SDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )";
+
+CONF_BASE_PATH="/etc/proxpn";
+CONF_FILE="proxpn.ovpn";
+configFilePath="$SDIR/$CONF_FILE"
+systemConfigFilePath="$CONF_BASE_PATH/$CONF_FILE";
+executableFileName="proxpn";
+executableFilePath="$SDIR/$executableFileName";
+systemInstallPath="/usr/local/bin";
+executableSystemInstallPath="$systemInstallPath/$executableFileName";
+CREDS_FILE="login.conf";
+systemCredsFilePath="$CONF_BASE_PATH/$CREDS_FILE";
+
+# Source: https://gist.github.com/hoylen/6607180
+PROG="$(basename "$0")";
+VERSION="0.2";
+
+#----------------------------------------------------------------
+# Process command line options
+
+HELP_TEXT="Usage: $PROG [options]
+Options:
+  -f | --force        Allow overwriting existing files on system if they already exist.  Useful for at least updating login info, and for also restoring program defaults.
+  -h | --help         show this help message";
+
+overwriteFiles="false";
+   
+while [ $# -gt 0 ]; do
+    case "$1" in
+        -h | --help)
+          echo "$HELP_TEXT";
+          exit 0;
+          ;;
+        -f | --force)
+          overwriteFiles="true";
+          ;;
+        --)
+          shift;
+          break;
+          ;; # end of options
+    esac
+    shift
+done
+
+#----------------------------------------------------------------
+# Main program flow
+
+if [ $EUID -ne 0 ]; then
+   echo "This script must be run as root in order to install system-wide configuration and program link.";
+   exit 1;
+fi
+
+if [[ ! -d "$CONF_BASE_PATH" ]]; then
+  mkdir "$CONF_BASE_PATH";
+fi
+
+## Install system-wide configuration, if not already installed.
+if [[ -f "$systemConfigFilePath" && ! "$overwriteFiles" == "true" ]]; then
+  echo "System-wide configuration file already exists; continuing installation...";
+elif [[ -f "$configFilePath" ]]; then
+  if cp "$configFilePath" "$systemConfigFilePath"; then
+    echo "Successfully installed system-wide configuration file.";
+  else
+    echo "Error: Not able to install system-wide configuration file; exiting...";
+    exit 1;
+  fi
+else
+  echo "Error: OpenVPN configuration file not found in project: \"$SDIR/$CONF_FILE\"";
+  echo "Exiting...";
+  exit 1;
+fi
+
+## Install system-wide program link, if not already installed.
+if [[ -f "$executableSystemInstallPath" && ! "$overwriteFiles" == "true" ]]; then
+  echo "System-wide program link already exists; continuing installation...";
+elif [[ -f "$executableFilePath" ]]; then
+  if [[ -f "$executableSystemInstallPath" ]]; then
+    rm "$executableSystemInstallPath";
+  fi
+  if ln -s "$(readlink -f "$executableFilePath")" "$executableSystemInstallPath"; then
```

Comment:
Understandable.

I will add a flag just in case this functionality is wanted, and use copy as the default method instead.